drivers/SmartThings/matter-appliance/src/matter-laundry/init.lua

```diff
@@ -51,11 +52,11 @@ local function device_init(driver, device)
 end
 
 -- Matter Handlers --
-local function is_matter_laundry_washer(opts, driver, device)
+local function is_matter_laundry_device(opts, driver, device)
   for _, ep in ipairs(device.endpoints) do
     for _, dt in ipairs(ep.device_types) do
-      if dt.device_type_id == LAUNDRY_WASHER_DEVICE_TYPE_ID then
-        return true
+      if dt.device_type_id == LAUNDRY_WASHER_DEVICE_TYPE_ID or dt.device_type_id == LAUNDRY_DRYER_DEVICE_TYPE_ID then
+        return dt.device_type_id
```

Comment:
Out of curiosity, why did this get changed to return `dt.device_type_id` instead 0f `true`?

Reply:
It doesn't seem to mean much. We can change it to true.

Reply:
Since the same sub-driver handles both laundry-dryer and laundry-washer devices, it is used to identify the device when switching profiles over [here](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/72cd6cc7c226a023b2ccf43cf1eb74b9c784e43e/drivers/SmartThings/matter-appliance/src/init.lua#L155-L162)

Reply:
ah, I see. Thank you!